src/Domains/Connectors/DriveCentric/Services/CustomerService.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\DriveCentric\Services;
+
+use Kanvas\Apps\Models\Apps;
+use Kanvas\Companies\Models\Companies;
+use Kanvas\Connectors\DriveCentric\Client;
+use Kanvas\Connectors\DriveCentric\Enums\ConfigurationEnum;
+
+class CustomerService
+{
+    public function __construct(
+        protected Companies $company,
+        protected Apps $app
+    ) {
+    }
+
+    public function getCustomers(int $offset = 0, string $start = 'today', ?string $endDate = null): array
+    {
+        $endDate = $endDate ?? date('Y-m-d');
+        $client = new Client($this->app);
```

Comment:
@FredPeal, move the client to the construct so you don't have to initialize it on each method, they are not static